main/.vuepress/config.js

```diff
@@ -36,47 +36,39 @@ module.exports = {
       'script',
       {},
       `
-    const fixups = new Map([
-      ['homeLink', () => {
-        //Anchor above the logo image
-        const homeEls = document.getElementsByClassName("home-link");
-        if(!homeEls.length) {
-          return false;
-        }
-        //Actual logo image
-        const homeEl = homeEls[0];
-        homeEl.setAttribute("href", "https://agoric.com");
-        homeEl.setAttribute("onclick", "return false;");
-        return true;
-      }],
-      ['logo', () => {
-        const logoEls = document.getElementsByClassName("logo")
-        if(!logoEls.length) {
-          return false;
-        }
-        const logoEl = logoEls[0]
-        logoEl.setAttribute("onclick", "document.location='https://agoric.com';return false;");
-        return true;
-      }],
-    ]);
+    /** @type {Map<[...anySelectors: string[]], (elems: (Element | null)[]) => unknown>} */
+    const fixups = new Map();
+
+    // Update the "home" link to target agoric.com while intercepting clicks
+    // such that those outside of its image continue routing to the root of
+    // the documentation site.
+    fixups.set(['.home-link', '.logo'], ([homeEl, logoEl]) => {
+      if (homeEl) {
+        homeEl.setAttribute('href', 'https://agoric.com');
+        homeEl.setAttribute('onclick', 'return false;');
+      }
+      if (logoEl) {
+        logoEl.setAttribute('onclick', "document.location='https://agoric.com';return false;");
+      }
+    });
```

Comment:
I was pretty sure this coupling of the `.home-link` fixup with the `.logo` fixup was brittle, not a feature.  Do you know anything that says otherwise?

Reply:
I think it's just robustness against future structural changes in Vuepress... if one selector matches but the other does not, we don't want the interval to continue firing and possibly matching the wrong content later on.